clang/test/CIR/CodeGen/complex-cast.c

```diff
@@ -0,0 +1,205 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir -mmlir --mlir-print-ir-before=cir-lowering-prepare -o %t.cir %s 2>&1 | FileCheck --check-prefixes=CIR-BEFORE,CHECK %s
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir -mmlir --mlir-print-ir-after=cir-lowering-prepare -o %t.cir %s 2>&1 | FileCheck --check-prefixes=CIR-AFTER,CHECK %s
```

Comment:
Please remove `CHECK`, just make `CIR-BEFORE`, `CIR-AFTER` and `LLVM` all check the labels

Reply:
Updated.